kms/attestations/verify_attestation.py

```diff
@@ -69,3 +70,17 @@ def verify(attestation_file, bundle_file):
                 continue
         return False
 # [END verify_attestations]
+
+
+if __name__ == '__main__':
```

Comment:
@kurtisvg Do we want to avoid adding new CLIs?

Reply:
Sounds good. Anything left?